cypher/cypher-prettifier/src/test/scala/org/neo4j/cypher/docgen/tooling/PrettifierParserTest.scala

```diff
@@ -56,7 +56,26 @@ class PrettifierParserTest extends ParserTestBase[Seq[SyntaxToken], Seq[SyntaxTo
         NonBreakingKeywords("assert"), AnyText("person.age"), NonBreakingKeywords("is unique"))
   }
 
-  test("shouldParseCreatIndexWithName") {
+  test("shouldParseCreateConstraintIfNotExists") {
+    // given
+    val query = "create constraint if not exists on (person:Person) assert person.age is unique"
+
+    // when then
+    parsing(query) shouldGive
+      Seq(BreakingKeywords("create constraint"), NonBreakingKeywords("if not exists"), BreakingKeywords("on"), GroupToken("(", ")",
+        Seq(AnyText("person:Person"))), NonBreakingKeywords("assert"), AnyText("person.age"), NonBreakingKeywords("is unique"))
+  }
+
+  test("shouldParseDropConstraintByName") {
```

Comment:
I'm guessing I decided to to add `CREATE OR REPLACE INDEX name`, `CREATE CONSTRAINT IF NOT EXISTS`, `DROP INDEX name IF EXISTS` and ``DROP CONSTRAINT name` before, and now only the last three is still valid, I think we want either `CREATE INDEX name` or `CREATE INDEX name IF NOT EXISTS` as well

(also in 4.2 then)